db/batchlog_manager.hh

```diff
@@ -89,12 +90,12 @@ private:
     cql3::query_processor& _qp;
     db_clock::duration _write_request_timeout;
     uint64_t _replay_rate;
-    timer<clock_type> _timer;
+    std::optional<future<>> _started;
```

Comment:
A simpler way is to use a non-optional future intialized to the coroutine, or to a ready future. But this is fine too.

Reply:
okay, will use a plain future<> initialized to a ready one.